dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -131,7 +131,7 @@ public void testBom()
       LinkageCheckerMain.main(new String[] {"-b", "com.google.cloud:libraries-bom:1.0.0"});
       fail("LinkageCheckerMain should throw LinkageCheckResultException upon errors");
     } catch (LinkageCheckResultException expected) {
-      assertEquals("Found 800 linkage errors", expected.getMessage());
+      assertEquals("Found 801 linkage errors", expected.getMessage());
```

Comment:
???

Reply:
LinkageCheckerMain groups by the targets of the linkage errors. As we're flipping the source and target, the number of groups changes.